### PR TITLE
Add blog post for rubygems adoptions

### PR DESCRIPTION
**Pull request by sonalkr132**
Related: https://github.com/rubygems/rubygems.org/pull/2748

[Rendered](https://medium.com/@sonalkr132/rubygems-adoptions-d09e50afe43)

### CODE REVIEW

File: Gemfile
Comment (jchestershopify) [inline Gemfile:2]: 2.7.4 has a trio of fairly serious CVEs against it. [I'd *strongly* recommend 2.7.5.](https://www.ruby-lang.org/en/news/2021/11/24/ruby-2-7-5-released/)

**Review by jchestershopify — APPROVED** [review]
Apart from the ruby version, LGTM.

File: Gemfile
Comment (jchestershopify) [reply Gemfile:2]: FWIW the same CVEs applied to all previous versions in the 2.6 and 3.0 lines as well, so 3.0.1 is also vulnerable -- 3.0.3 is the patched version in the 3.0 line.

Comment (jchestershopify) [reply Gemfile:2]: Ah, just realised this isn't the .org repo, it's the blog 🤦

Comment (sonalkr132) [reply Gemfile]: thanks. I changed it to 2.7.5. It is related to this https://github.com/rubygems/rubygems.github.io/pull/78#issuecomment-983525018

File: _posts/2021-12-01-rubygems-adoptions.md
Comment (deivid-rodriguez) [inline _posts/2021-12-01-rubygems-adoptions.md:28]: I think this line is not intended?

```suggestion
```

**Review by deivid-rodriguez — APPROVED** [review]
This is really awesome @sonalkr132. Thanks for all you do for the community! ❤️

File: _posts/2021-12-01-rubygems-adoptions.md
Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: Note that "ownership_call" is the name of the model. We are trying to communicate that the user is asking (calling) for new ownerships. I agree that we are trying to invent a term here and this section attempts to explain what "ownership calls" mean.

> any reason to not use more common terminology like "Looking for maintainer"

AFAIU, model names are almost always nouns. I am not sure pluralization would work for "looking_for_maintainer" in the context of the rails association. Further, we are looking for two model names, one created by owners and the other by users. Preferably they should use similar terms. Can you please suggest what could the name for objects created by users in response to "looking for maintainer"?
Are you suggesting we use "looking for maintainer" only for the blog?

> TBH I had no idea what this section is about

Could you please elaborate with examples? I am hoping they are grammatically correct and coherent sentences. It is not very helpful to just say "I have no idea". What changes would you make?

Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: ping

Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: ping

Comment (simi) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: Hello, sorry for the delay. We can call that section "Looking for the maintainer" or something and keep the models the same. I think there is no need to keep the model and section names the same.

I think similar problem was raised at https://github.com/rubygems/rubygems.org/issues/2892.

Comment (simi) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: > TBH I had no idea what this section is about. 

I was referring to the fact when I open the page, I have no idea what's the purpose thanks to used naming. That's my comment as non English native speaker. Maybe I have just some language gap, that's also possible.

Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: FYI, I have updated the view to address some of your concerns here https://github.com/rubygems/rubygems.org/pull/2910. I am sticking with ownership call/request naming for view with added explanation in subtitle and text area placeholder. I have added "looking for maintainer" as a tooltip. I would appreciate it if you could share your concerns or suggestions before it gets merged.

Comment (sonalkr132) [reply _posts/2021-12-01-rubygems-adoptions.md:14]: reminder to review the proposed PR so that we avoid arguments after released.

**Comment by sonalkr132** [comment]
Hi, I am moving ahead with this without confirmation from @simi on whether changes made to the adoption page addressed his concern. I have waited almost a month.

**Comment by simi** [comment]
Sorry :( I had no time to write down my notes.

**Comment by sonalkr132** [comment]
Please create a new issue with any further suggestions you may have.

**Pull request merged**
